### PR TITLE
sql-function-cos

### LINKED ISSUE #2262: [Term Entry] SQL Math Functions COS
### Reason/inspiration (optional)

We would like a new term entry for the Math Function, `COS`, in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/terms/cos/cos.md

The entry should: 

- Explain what the math function `COS` is in SQL and what it’s used for, starting with an introductory paragraph (2-3 sentences).
- Include SQL database compatibility as a note.
- Links out to entries as needed.

Please refer to the [entry templates (term entry template in particular)](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Thanks for assigning. This closes #2262 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/cos/cos.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The SQL function **`COS()`** calculates the cosine of a given angle, expressed in radians. It returns the ratio of the adjacent side of a right triangle to its hypotenuse, representing the cosine value between -1 and 1.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/cos/cos.md:18]:
```suggestion
The SQL function **`COS()`** calculates the cosine of a given angle, expressed in radians.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I just added a few comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by nikolaspoczekaj** [comment]
@caupolicandiaz thanks for the review! I'm done with the changes.

**Comment by nikolaspoczekaj** [comment]
cant really tell why the format verify is failing...

**Pull request merged**